TestStack.Dossier/TestDataBuilder.cs

```diff
@@ -71,10 +72,22 @@ public TObject Build()
         }
 
         /// <summary>
-        /// Build the actual object - override this and call the constructor and any other methods.
+        /// Build the actual object - you can optionally override this and call the constructor and any other methods.
         /// </summary>
         /// <returns>The built object</returns>
-        protected abstract TObject BuildObject();
+        protected virtual TObject BuildObject()
+        {
+            var model = Any.Fixture.Create<TObject>();
```

Comment:
This can't be the default BuildObject implementation because if you have ctor params, but not public setters then the values you have customised won't be used right?

I think we have to find the ctor ourself and then pull the values from Get if they match up to properties on the class (and if not then pull an anonymous value (from Autofixture maybe)?